lib/chunked_uploader.dart

```diff
@@ -40,7 +43,7 @@ class ChunkedUploader {
         maxChunkSize: maxChunkSize,
         onUploadProgress: onUploadProgress,
         headersCallback: headersCallback,
-      ).upload();
+      ).upload(stream, queryParameters);
```

Comment:
Maybe it's better to pass in the `queryParameters` in `_Uploader` constructor and change `stream` to `multipart` which indicates if request data should be multipart or not. What do you think?

Reply:
Yep! Will do it ASAP!